### DISCUSSION TITLE
Noob with v4 - Think I have a specific issue with HideMyAss VPN

### DISCUSSION THREAD

**Discussion by ScareDcrow2** [haugene/docker-transmission-openvpn#2007] [Q&A] [answered]
Here's the output on the console...doesn't seem to matter whether I use correct or dummy credentials so I don't think it proceeds that far

Mon Oct 11 20:48:36 2021 WARNING: --ns-cert-type is DEPRECATED.  Use --remote-cert-tls instead.
Mon Oct 11 20:48:36 2021 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Mon Oct 11 20:48:36 2021 TCP/UDP: Preserving recently used remote address: [AF_INET]199.189.106.245:443
Mon Oct 11 20:48:36 2021 Socket Buffers: R=[16777216->16777216] S=[16777216->16777216]
Mon Oct 11 20:48:36 2021 Attempting to establish TCP connection with [AF_INET]199.189.106.245:443 [nonblock]
Mon Oct 11 20:48:37 2021 TCP connection established with [AF_INET]199.189.106.245:443
Mon Oct 11 20:48:37 2021 TCP_CLIENT link local: (not bound)
Mon Oct 11 20:48:37 2021 TCP_CLIENT link remote: [AF_INET]199.189.106.245:443
Mon Oct 11 20:48:37 2021 Connection reset, restarting [0]
Mon Oct 11 20:48:37 2021 SIGTERM[soft,connection-reset] received, process exiting

and my docker-compose config is like

      - OPENVPN_PROVIDER=HIDEMYASS
      - OPENVPN_CONFIG=Canada.BritishColumbia.Vancouver.TCP,USA.Maine.Bath.TCP,USA.Utah.SaltLakeCity.TCP
      - OPENVPN_USERNAME=dummy
      - OPENVPN_PASSWORD=password

**Comment by pkishino** [comment, marked as answer]
Suspect outdated vpn profiles.. next step would be to download directly from provider and testing using CUSTOM as provider and mounting the profile as per documentation

**Reply by ScareDcrow2** [reply]
Thanks so much. Yes that solved it. I then had an issue that I figured out was that I needed to turn on IPv6. For anyone searching, the error was

Tue Oct 12 09:42:46 2021 /sbin/ip -6 addr add 2001:db8:123::2/64 dev tun0
RTNETLINK answers: Permission denied
Tue Oct 12 09:42:46 2021 Linux ip -6 addr add failed: external program exited with error status: 2
Tue Oct 12 09:42:46 2021 Exiting due to fatal error

and the fix was to add this to the docker compose

    sysctls:
      - "net.ipv6.conf.all.disable_ipv6=0"
